Tools/ardupilotwaf/boards.py

```diff
@@ -368,7 +368,7 @@ def configure_env(self, cfg, env):
             '-mthumb',
             '-DCHIBIOS_BOARD_NAME="%s"' % self.name,
             '--specs=nano.specs',
-            '-specs=nosys.specs'
+            '--specs=nosys.specs'
```

Comment:
This seems wrong, according to GCC manual, the flag is `-specs=`. Did you test this?

Reply:
all documents beside gcc one's write --specs

Reply:
Not _all documents_... Some places do show the double dash, which is probably why GCC started accepting it. Manual still uses the single dash so that's what we should use.

Unless you want to change the PR to make the only double-dash to single-dash, I'll close this.

Reply:
ok I am closing!